src/wrapper.ts

```diff
@@ -6,109 +6,63 @@
  * Copyright 2019 Datadog, Inc.
  */
 
-import fs from "fs";
-import path from "path";
 import Service from "serverless/classes/Service";
-import util from "util";
 import { FunctionInfo, RuntimeType } from "./layer";
-import { es6Template } from "./templates/node-es6-template";
-import { nodeTemplate } from "./templates/node-js-template";
-import { typescriptTemplate } from "./templates/node-ts-template";
-import { pythonTemplate } from "./templates/python-template";
-import { getHandlerPath, removeDirectory } from "./util";
-import { TracingMode } from "./templates/common";
 
-export const datadogDirectory = "datadog_handlers";
+export const datadogHandlerEnvVar = "DD_LAMBDA_HANDLER";
+export const pythonHandler = "datadog_lambda.handler.handler";
+export const pythonHandlerFile = "datadog_lambda.handler.py";
+export const jsHandlerLayerPrefix = "/opt/nodejs/";
+export const jsHandler = "node_modules/datadog-lambda-js/handler.handler";
+export const jsHandlerFile = "node_modules/datadog-lambda-js/handler.js";
```

Comment:
Not sure on the path here, when I install the package with `npm`, my local path is `node_modules/datadog-lambda-js/dist/hander.js`, but after the changes of moving `redirectHandlers` into `afterPackageFunction`, no error is generated for either version of the file path.